pkg/apis/build/v1alpha1/image_builds.go

```diff
@@ -28,6 +28,7 @@ type AbstractBuilder interface {
 	ImageRef() BuilderImage
 	Ready() bool
 	BuildpackMetadata() BuildpackMetadataList
+	GetName() string
```

Comment:
Super pedantic but, get Getters in go shouldn't start with 'Get`.

https://golang.org/doc/effective_go.html#Getters